cookbook/tools/discord_tools.py

```diff
@@ -0,0 +1,43 @@
+"""Run `pip install discord.py` to install dependencies."""
+
+from phi.agent import Agent
+from phi.tools.discord_tools import DiscordTools
+
+# Export the DISCORD_BOT_TOKEN environment variable or provide it directly
+# - DISCORD_BOT_TOKEN
+
+# Initialize the Discord toolkit
+discord_tools = DiscordTools(
+    bot_token="your-bot-token",  # Replace with your Discord bot token
+    enable_messaging=True,
+    enable_history=True,
+    enable_channel_management=True,
+)
+
+# Create an agent with the discord toolkit
+agent = Agent(
+    instructions=[
```

Comment:
Add a unique name and pass model to the agent, also set markdown to true.
here is an example - https://github.com/phidatahq/phidata/blob/main/cookbook/agents_101/web_search.py

Reply:
solved for this too